index.html

```diff
@@ -57,9 +57,6 @@
                         <p class="text-intro">
                             TrickFire Robotics is a student-led robotics team focused on providing a 
                             unique educational experience through hands-on work on large scale projects. 
-                            Since 2016 the team has traveled to the Kennedy Space Center to represent the 
```

Comment:
We should add another sentence. The homepage feels too empty. Perhaps say that TrickFire competes in University-level competitions, with the NASA RMC in the past and the URC in the present.